nemo/utils/export_utils.py

```diff
@@ -100,23 +100,24 @@ def parse_input_example(input_example):
     return input_list, input_dict
 
 
-def to_onnxrt_input(input_names, input_list, input_dict):
+def to_onnxrt_input(input_names, input_dict, input_list):
     odict = {}
-    for k, v in input_dict.items():
-        odict[k] = v.cpu().numpy()
-    for i, input in enumerate(input_list):
-        if type(input) in (list, tuple):
-            odict[input_names[i]] = tuple([ip.cpu().numpy() for ip in input])
+    for k in reversed(input_names):
```

Comment:
any reason for using reversed?

Reply:
For the theoretical use case when both input_list and input_dict are present - that would mean, by export() API rules,  the dict is the last, so we need to start from the back.